tests/TestCase/View/Helper/FlashHelperTest.php

```diff
@@ -15,6 +14,15 @@
  */
 class FlashHelperTest extends TestCase
 {
+    /**
+     * @var View
+     */
+    private $View;
```

Comment:
We don't use privates and protected would need the var name to be prefixed with underscore. So just change all to `public`.